lib/ttnt/testtask.rb

```diff
@@ -22,9 +30,19 @@ def initialize(rake_testtask)
 
       @anchor_description = 'Generate test-to-code mapping' + (@rake_testtask.name == :test ? '' : " for #{@rake_testtask.name}")
       @run_description = 'Run selected tests' + (@rake_testtask.name == :test ? '' : "for #{@rake_testtask.name}")
+
+      @@instance_by_name[@rake_testtask.name.to_s] = self
       define_tasks
     end
 
+    # Returns array of test file names.
+    #   Unlike Rake::TestTask#file_list, patterns are expanded.
+    def expanded_file_list
+      test_files = Rake::FileList[@rake_testtask.pattern].compact
+      test_files += @test_files.to_a if @test_files
```

Comment:
Can't this method simply be defined as:

~~~ruby
def expanded_file_list
  Rake::FileList[@rake_testtask.pattern].compact
end
~~~

? At least the suite is passing with such method so this is either useless or you are missing a test where this line would be useful.